CMakeExt/MPI.cmake

```diff
@@ -30,9 +30,18 @@ if (MPI_INCLUDE_PATH AND MPI_LIBRARY)
   elseif ("${MPI_INCLUDE_PATH}" MATCHES "openmpi")
```

Comment:
You could improve the Open MPI detection, by replacing this string matching with a compile check which tests for the `OPEN_MPI` or the version macros.